openrpc/src/stellar-rpc/schemas/Ledgers.json

```
@@ -0,0 +1,29 @@
+{
+  "Ledgers": {
+    "type": "object",
+    "properties": {
+      "hash": {
+        "type": "string",
+        "description": "The ledger hash."
+      },
+      "sequence": {
+        "type": "number",
+        "description": "The sequence number of the ledger.",
```

Suggestion, for people coming from other changes
```suggestion
        "description": "The sequence number of the ledger (sometimes called the "block height").",
```